### PR TITLE
Explain use of quotes around code fragments

### LINKED ISSUE #3191: How to mark short inline code listings
As noted we now have single quotes around small language fragments; as far as I can see it was added for some in 
https://github.com/modelica/ModelicaSpecification/pull/2726 in a commit named "Migrate remaining glossary entries".

I understand there might be some need for disambiguating small Modelica-fragments from non-Modelica code in the rendered text, but I don't see that there were any discussion about this change, and I also see suggestions for using special markings for spaces indicating that it doesn't suffice.

The issues I see with this solution are:

- It is not automated.
- There are no 100% clear rules when it applies; the ideal would be to apply the same rule to all for inline fragments - but having single quotes around all inline code fragments would be too much.
- (A less important issue is that we cannot tell Modelica and grammar-fragments apart.)

The most obvious alternative would seem to be to have a slightly different background color for all inline fragments (think a whiter shader of lightgray - not yellow), but lstinline does not support that out of the box. However it seems possible: https://tex.stackexchange.com/questions/30845/how-to-redefine-lstinline-to-automatically-highlight-or-draw-frames-around-all/30851 (except that it fails for line-breaks).

### PR DESCRIPTION
**Pull request by henrikt-ma**
According to phone meeting decision: https://github.com/modelica/ModelicaSpecification/issues/3191#issuecomment-1351689447

Fixes #3191.

### CODE REVIEW

File: chapters/arrays.tex [hunk 1]
```diff
@@ -1263,7 +1263,7 @@ \subsection{Element-wise Division}\label{array-element-wise-division}\label{elem
 2./[1, 2; 3, 4]  // error; same as  2.0 / [1, 2; 3, 4]
 2 ./[1, 2; 3, 4] // fine; element-wise division
 \end{lstlisting}
-This is a consequence of the parsing rules, since `\lstinline!2.!' is a lexical unit.  Using a space after the literal solves the problem.
+This is a consequence of the parsing rules, since ``\lstinline!2.!'' is a lexical unit.  Using a space after the literal solves the problem.
```
Comment (HansOlsson) [inline chapters/arrays.tex:1266]: I would prefer to just have single quotes for short code fragments, regardless of whether they are one character or a few - i.e. skip this.

Comment (henrikt-ma) [reply chapters/arrays.tex:1266]: I'm not against it.  You do realize that his means there are a few other places where we'll then need to change from double to single quotes?

Comment (HansOlsson) [reply chapters/arrays.tex:1266]: Yes. But better to have one rule, and similarly as we say that we can add the quotes later when needed we can also clean up that later.

Comment (henrikt-ma) [reply chapters/arrays.tex]: OK now?

**Review by HansOlsson — APPROVED** [review]
Ok

**Pull request merged**